### PR TITLE
Optimize graphs #4180

### LINKED ISSUE #4180: Laggy Debbuger
https://user-images.githubusercontent.com/90691415/152486591-0afb5464-e29d-45b3-9ca3-292589947f37.mov

### PR DESCRIPTION
**Pull request by Norbiros**
Resolves #4180

### Changes

Create new option, hide fill and animation for smaller lags.

### Reason for changes

Some people have lags viewing graphs.

### Tests

I tested it in browser :)

### CODE REVIEW

File: addons/debugger/addon.json [hunk 1]
```diff
@@ -73,23 +73,23 @@
...
     }
   ],
   "tags": ["editor", "featured"],
   "enabledByDefault": false,
   "versionAdded": "1.16.0",
   "latestUpdate": {
-    "version": "1.24.0",
-    "isMajor": true,
-    "temporaryNotice": "New \"Threads\" and \"Performance\" tabs in the debugger window.",
+    "version": "1.24.1",
```
Comment (RedGuy12) [inline addons/debugger/addon.json:88]: 1.25.0

File: addons/debugger/addon.json [hunk 2]
```diff
@@ -73,23 +73,23 @@
...
+    "isMajor": false,
+    "temporaryNotice": "New option to make graphs not laggy!",
```
Comment (RedGuy12) [inline addons/debugger/addon.json:90]: not needed

Comment (WorldLanguages) [reply addons/debugger/addon.json:90]: Agreed, the change is too small to warrant a lastUpdated change.

**Comment by apple502j** [comment]
Why have optimization a setting? Why not turn it on always?

**Comment by Norbiros** [comment]
Because it looks no so pretty... It is line without any animation. We can change it... but default in previous version was with fill etc, so i think let's keep as it is. 
Also i found some bugs with dark mode... i will fix it tomorrow?

**Can someone tell me if in line optimization mode, must be gray, maybe change it to blue and add some thickness?**

**Comment by apple502j** [comment]
@Norbiros Then the setting should be renamed to "Use fancy graph" (default on) or "Faster graph rendering" (default off). "Optimization" sounds like there is no compromise.

**Comment by Norbiros** [comment]
About this error. Can someone check it again? I think I fixed error with `Format code using Prettier`.

**Pull request closed without merging**